src/niri.rs

```diff
@@ -299,6 +299,11 @@ struct SurfaceFrameThrottlingState {
     last_sent_at: RefCell<Option<(Output, u32)>>,
 }
 
+pub enum CenteringMode {
+    CoordsIndividually,
+    IfNeeded,
```

Comment:
I see what you were going for with naming here, but the result is kinda weird. I suggest changing this to something like `CenterCoords::{Separately, Both}`.

Reply:
fixed in commit [123c2f0](https://github.com/YaLTeR/niri/pull/245/commits/123c2f040e1891c7ce5d3d7c29d497f48da788cd)